From: Federico Ceratto <federico.ceratto@gmail.com>
Date: Fri, 2 Jan 2015 01:46:17 +0000
Subject: Add setup.py

---
 setup.py | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)
 create mode 100644 setup.py

diff --git a/setup.py b/setup.py
new file mode 100644
index 0000000..fffb757
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+readme = open("README.md", "r")
+
+setup(name='pyroute2',
+      version='@RELEASE@',
+      description='Python Netlink library',
+      author='Peter V. Saveliev',
+      author_email='peter@svinota.eu',
+      url='https://github.com/svinota/pyroute2',
+      license='GPLv2+',
+      packages=['pyroute2',
+                'pyroute2.ipdb',
+                'pyroute2.netlink',
+                'pyroute2.netlink.generic',
+                'pyroute2.netlink.ipq',
+                'pyroute2.netlink.nfnetlink',
+                'pyroute2.netlink.rtnl',
+                'pyroute2.netlink.taskstats'],
+      classifiers=['License :: OSI Approved :: GNU General Public ' +
+                   'License v2 or later (GPLv2+)',
+                   'Programming Language :: Python',
+                   'Topic :: Software Development :: Libraries :: ' +
+                   'Python Modules',
+                   'Operating System :: POSIX',
+                   'Intended Audience :: Developers',
+                   'Development Status :: 4 - Beta'],
+      long_description=readme.read())
